pipeline_live/data/sources/alpaca.py

```diff
@@ -27,8 +28,10 @@ def _get_stockprices(symbols, limit=365, timespan='day'):
     Just deal with Alpaca's 200 stocks per request limit.
     '''
 
+    api = tradeapi.REST()
+
     def fetch(symbols):
-        barset = tradeapi.REST().get_barset(symbols, timespan, limit)
+        barset = api.get_barset(symbols, timespan, limit)
```

Comment:
this looks better. now we have to make sure it works as expected, and that the warning you faced is solved.